tests/protractor/page-objects/users/users.po.js

```diff
@@ -0,0 +1,35 @@
+var helper = require('../../helper');
+var addUserButton = element(by.id('add-user'));
+//var addUserModal = $('mm-modal#edit-user-profile');
+var usersList = element.all(by.repeater('user in users'));
+
+module.exports = {
+	addUser: function (username, password) {
+		username.trim();
+		password.trim();
+		helper.waitUntilReady(addUserButton);
+	},
+
+	editUser: function (username, password) {
+		username.trim();
+		password.trim();
+		helper.waitUntilReady(addUserButton);
+	},
+
+	deleteUser: function (username, password) {
+		username.trim();
+		password.trim();
+		helper.waitUntilReady(addUserButton);
+	},
+
+	openAddUserModal: function () {
+		helper.waitElementToBeClickable(addUserButton);
+		addUserButton.click();
+	},
+	getUsersList:function(){
+		helper.waitUntilReady(addUserButton);
+		return usersList;
+	}
+};
```

Comment:
The indentation in this file is all wrong. Make sure you editor is set to 2 spaces.